marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -245,9 +245,85 @@ void testRowExport() throws Exception {
             });
     }
 
-    private void assertRowsAndColumns(int limit, int totalColumns, String response) {
+    @Test
+    void testRowExportWithPathRangeQuery() throws Exception {
+        runAsDataHubDeveloper();
+        ReferenceModelProject project = installOnlyReferenceModelEntities(true);
+        deployEntityIndexes(getHubConfig());
+
+        Customer customer1 = new Customer();
+        customer1.setCustomerId(1);
+        customer1.setName("Jane");
+        customer1.setCustomerNumber(123456789);
+        customer1.setCustomerSince("2012-05-16");
+        project.createCustomerInstance(customer1);
+
+        Customer customer2 = new Customer();
+        customer2.setCustomerId(2);
+        customer2.setName("John");
+        customer2.setCustomerNumber(987654321);
+        customer2.setCustomerSince("2013-06-16");
+        project.createCustomerInstance(customer2);
+
+        String json = "{\n" +
+                "    \"savedQuery\": {\n" +
```

Comment:
@rahulvudutala  @akshaysonvane We've got several instances of inline JSON like this. Thoughts on simplifying this? I see a maintenance burden here where if we want to change something about this data structure, we'll need to update all of these tests. Ideally, @ryanjdew  can obtain a saved query here without having to copy/paste one from another test. (This would be a future cleanup, not something to address in this ticket)